docs/contributing.md

```diff
@@ -0,0 +1,32 @@
+# Contributing
+
+We welcome contributions to this project!
+
+There are many ways to contribute! We welcome all contributions including:
+
+* File an Issue
+* Request a Feature
+* Implement a Requested Feature
+* Fix an Issue/Bug
+* Add/Fix documentation
+
+Contributions must follow the guidelines outlined on the [Tableau Organization](http://tableau.github.io/) page, though filing an issue or requesting
+a feature do not require the CLA.
+
+## Issues and Feature Requests
+
+To submit an issue/bug report, or to request a feature, please submit a [github issue](https://github.com/tableau/document-api-python/issues) to the repo.
+
+If you are submiting a bug report, please provide as much information as you can, including clear and concise repro steps, attaching any necessary
+files to assist in the repro.  **Be sure to scrub the files of any potentially sensitive information.  Issues are public.**
+
+For a feature request, please try to describe the scenario you are trying to accomplish that requires the feature.  This will help us understand
+the limitations that you are running into, and provide us with a use case to know if we've satisfied your request. Psuedo code that describes how you would like things to work is also very helpful.
+
+## Fixes, Implementations, and Documentation
+
+For all other things, please submit a PR that includes the fix, documentation, or new code that you are trying to contribute.  More information on
+creating a PR can be found in the [github documentation](https://help.github.com/articles/creating-a-pull-request/)
+
+If the feature is complex or has multiple solutions that could be equally appropriate approaches, it would be helpful to file an issue to discuss the
+design trade-offs of each solution before implementing, to allow us to collectively arrive at the best solution, which most likely exists in the middle somewhere.
```

Comment:
Couldn't this be a symlink to the upper-level contributing.md (I think we have one)